tcpdf.php

```diff
@@ -7779,16 +7777,16 @@ public function _destroy($destroyall=false, $preserve_objcopy=false) {
 			// remove all temporary files
 			if ($handle = @opendir(K_PATH_CACHE)) {
 				while ( false !== ( $file_name = readdir( $handle ) ) ) {
-					if (strpos($file_name, '__tcpdf_'.$this->file_id.'_') === 0) {
+					if (strpos($file_name, '__tcpdf_'.$this->file_id.'_') === 0 && file_exists(K_PATH_CACHE.$file_name)) {
```

Comment:
file_exists is a security issue.

Reply:
Could you explain why, it would be interesting :)